keras_cv/models/segmentation/fcn_test.py

```diff
@@ -0,0 +1,205 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+
+import pytest
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv import models
+from keras_cv.models.segmentation import FCN
+
+
+class FCNTest(tf.test.TestCase):
+    def test_fcn_model_with_vgg16_backbone_construction_with_preconfigured_setting(
+        self,
+    ):
+        model = FCN(
+            classes=11,
+            backbone="vgg16",
+            model_architecture="fcn8s",
+            input_shape=(256, 256, 3),
+        )
+        input_image = tf.random.uniform(shape=[2, 256, 256, 3])
+        output = model(input_image)
+
+        self.assertEquals(output.shape, [2, 256, 256, 11])
+
+    def test_fcn_model_with_vgg19_backbone_construction_with_preconfigured_setting(
+        self,
+    ):
+        model = FCN(
+            classes=11,
+            backbone="vgg19",
+            model_architecture="fcn8s",
+            input_shape=(256, 256, 3),
+        )
+        input_image = tf.random.uniform(shape=[2, 256, 256, 3])
+        output = model(input_image)
+
+        self.assertEquals(output.shape, [2, 256, 256, 11])
+
+    def test_fcn_model_with_vgg16_components(self):
+        backbone = models.VGG16(include_rescaling=False, include_top=False)
```

Comment:
Okay. I am now incorporating a change which will identify if a KCV VGG16 or 19 model has been passed as an argument. If yes, it will then process is accordingly as it would the string arguments